website/pages/en/index.js

```diff
@@ -135,38 +135,32 @@ module.exports = class Index extends React.Component {
             <h2 className="homepage__feature-list__title">What Richie brings you</h2>
             <div className="blockElement blockElement--left twoByGridBlock">
               <div className="blockContent">
-                <h3>An LMS-agnostic education portal</h3>
+                <h3>An LMS-agnostic Education Portal</h3>
                 <p>
-                  Your course catalog can be synchronized with one or more LMS instances running
-                  different kinds of software, such as Open edX or Moodle. Richie is built to
-                  aggregate it all for your users.
+                  Course catalogs can synchronize with one or more LMS instances running different software, 
+                  such as Open edX or Moodle. Richie aggregates it all for your users.
                 </p>
-                <h3>An editor back-office</h3>
+                <h3>Author-first</h3>
                 <p>
-                  All the content in Richie is meant to be created and updated by regular editors,
-                  not software engineers. As it is based on Django CMS, you get a rich editor
-                  interface along with Richie.
+                  Content authors do not have to rely on software engineers to create and update all materials in Richie. 
+                  Instead, authors use a rich editor interface to maintain content.
```

Comment:
should we mention "frontend editing"?

Reply:
It depends on the audience. If the "selling" point is that anyone unfamiliar with code can use Richie, mentioning "frontend editing" may be confusing.

Reply:
ok then. Let's leave it like this :+1: